routes/lecture.js

```diff
@@ -0,0 +1,103 @@
+const { Lecture, validate } = require("../models/Lecture")
+const router = require("express").Router()
+const teacherAuth = require("../middlewares/teacherAuth")
+const auth = require("../middlewares/auth")
+const jwt = require("jsonwebtoken")
+const _ = require("lodash")
+
+router.get(
+  "/",
+  () => auth(req, res, next, false),
```

Comment:
the middleware will get req,res,next , pass that to custom middleware

Reply:
okay